test/tests.config.json

```diff
@@ -18,7 +18,7 @@
     "/scroll-animations/css/at-scroll-timeline-cascade.html",
     "/scroll-animations/css/at-scroll-timeline-element-offsets.html",
     "/scroll-animations/css/at-scroll-timeline-inactive-phase.html",
-    "/scroll-animations/css/at-scroll-timeline-orientation.html",
+    "/scroll-animations/css/at-scroll-timeline-axis.html",
```

Comment:
This file should be synced from WPT.  None of the .../css/at-scroll-* tests exist any longer.

Reply:
I will make RP for WPT.

Reply:
What I'm getting at is that all of the at-scroll-timeline-* tests have been removed from wpt as they were testing obsolete syntax.  So we shouldn't update to exclude a non-existent test, but rather ensure that the json file reflects tests that are actually in wpt.  Also, updating test.config.json affects anyone testing this polyfill.  It may make sense to keep your own modified version of the config file to restrict testing to certain tests, but we'll want test coverage even for those that fail.  I believe these exclusions were added at a time when the spec was in flux and we were phasing out some old syntax.